src/composables/queries/useGraphQuery.ts

```diff
@@ -63,5 +62,5 @@ export default function useGraphQuery<T>(
     }
   };
 
-  return useQuery(queryKey, queryFn, options);
+  return useQuery(queryKey, queryFn, options as any);
```

Comment:
I did not find a way to make TS compiling the options type 🤷‍♂️
Maybe you can give it a try 🧙

Reply:
How's this:
```suggestion
  return useQuery<T>(queryKey, queryFn, options);
```

Reply:
I forgot that useQuery was expecting a generic.  Thanks!